techniques/html/H36.html

```diff
@@ -2,108 +2,102 @@
+            <li><a href="#important-information">達成方法に関する重要な情報</a></li>
+            <li><a href="#applicability">適用 (対象)</a></li>
+            <li><a href="#description">解説</a></li>
+            <li><a href="#examples">事例</a></li>
+            <li><a href="#related">関連する達成方法</a></li>
+            <li><a href="#tests">検証</a></li>
          </ul>
       </nav>
-      <h1>Using <code class="att">alt</code> attributes on images used as submit buttons
-      </h1>
+      <h1>送信ボタンとして用いる画像の <code class="att">alt</code> 属性を使用する</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p> Applies to content using image-based submit buttons. </p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/non-text-content">Success Criterion 1.1.1: Non-text Content</a> (Sufficient)</span>.
+            <h2>適用 (対象)</h2>
+            <p> 画像の送信/実行ボタンを使用するコンテンツに適用する。</p>
+            <p>これは<span><a href="https://waic.jp/docs/WCAG21/Understanding/non-text-content">Success Criterion 1.1.1: 非テキストコンテンツ</a> (十分)</span> に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p> For input elements of type 'image', the <code class="att">alt</code> attribute of the
-               <code class="el">input</code> element is used to provide a functional label. This label indicates
-               the button's function, but does not attempt to describe the image. The label is
-               especially important if there are multiple submit buttons on the page that each lead
-               to
-               different results. 
+            <h2>解説</h2>
+            <p>この達成方法の目的は、type 属性が「image」である input 要素において、<code class="el">input</code> 要素の <code class="att">alt</code> 属性を機能的なラベルを提供するのに使用することである。このラベルは、画像の説明をするのではなく、ボタンの機能を説明する。もしそのページに、それぞれ異なる結果につながる複数の送信/実行ボタンがあるならば、ラベルは特に重要である。
```

Comment:
ほぼ趣味みたいなものですが、かぎ括弧を原語に寄せます。
```suggestion
            <p>この達成方法の目的は、type 属性が 'image' である input 要素において、<code class="el">input</code> 要素の <code class="att">alt</code> 属性を機能的なラベルを提供するのに使用することである。このラベルは、画像の説明をするのではなく、ボタンの機能を説明する。もしそのページに、それぞれ異なる結果につながる複数の送信/実行ボタンがあるならば、ラベルは特に重要である。
```